CHANGELOG.md

```diff
@@ -4,6 +4,42 @@
 
 [1]: https://www.npmjs.com/package/nodejs-pubsub?activeTab=versions
 
+## [0.29.0](https://www.github.com/googleapis/nodejs-pubsub/compare/v0.28.1...v0.29.0) (2019-05-15)
```

Comment:
Is there a reason *not* to go 1.0?

Reply:
Old habits die hard? I think @ajaaym and @sduskis are pretty close to closing the GA issues, so I assumed we'd want to wait for that. @sduskis have any feelings here?